.github/workflows/build.yml

```diff
@@ -46,9 +46,12 @@ jobs:
 
       - name: Set up Java  ${{ env.JAVA_VERSION }}
         if: env.NODE_VERSION == '14'
-        uses: actions/setup-java@v1.4.3
+        uses: actions/setup-java@v2.0.0
         with:
           java-version: ${{ env.JAVA_VERSION }}
+          distribution: 'adopt'
+          java-package: jdk
```

Comment:
isn't jre enough?

Reply:
I don't know, maybe @rarkins knows?

The "old" `v1` version of the `actions/setup-java` uses the JDK according to the action run logs. But maybe the JRE is enough to run the tests. I don't know if we need any of the extra stuff that comes with the JDK.

Reply:
I thought JRE should be enough. Let's try

Reply:
I've pushed a commit that sets this to `jre`.